Makefile

```diff
@@ -653,6 +658,20 @@ verify-tiltfile: ## Verify Tiltfile format
 verify-container-images: ## Verify container images
 	TRACE=$(TRACE) ./hack/verify-container-images.sh
 
+.PHONY: verify-govulncheck
+verify-govulncheck: $(GOVULNCHECK) ## Verify code for vulnerabilities
+	$(GOVULNCHECK) ./... && R1=$$? || R1=$$?; \
+	$(GOVULNCHECK) -C "$(TOOLS_DIR)" ./... && R2=$$? || R2=$$?; \
+	$(GOVULNCHECK) -C "$(TEST_DIR)" ./... && R3=$$? || R3=$$?; \
+	if [ "$$R1" -ne "0" ] || [ "$$R2" -ne "0" ] || [ "$$R3" -ne "0" ]; then \
+		exit 1; \
+	fi
```

Comment:
@sbueringer : Not sure if we should do it like this or move this to a bash script...

Reply:
Let's also put this in a shell script like we did for verify-container-images

Reply:
But maybe we can get rid of the verify-security.sh script? Do you think it's relatively simple to call the two other shell scripts in verify-security?

Reply:
That would have the con of not being able to only run govulncheck for all modules.

But I'm ok with that.

Reply:
Jep should be easy 👍